app/server/schemas.py

```diff
@@ -84,11 +84,20 @@ def get_profile_url(self, obj):
         return processed_profile_pictures
 
 
-class UploadedImageSchema(Schema):
-    id                      = fields.Int(dump_only=True)
-    filename                = fields.Str()
-    image_url               = fields.Function(lambda obj: obj.image_url)
-    credit_transfer_id      = fields.Int()
+class UploadedResourceSchema(Schema):
```

Comment:
oh whoa i didn't see this file -- are we supposed to manually create an entry into it? my ussd tables didn't automatically generate this. and where does this get used?

Reply:
this gets used when we want to serialise a SQLAlchemy object into a JSON string. For example in user_api:

`            response_object = {
                'status': 'success',
                'message': 'Successfully Loaded.',
                'data': {
                    'user': user_schema.dump(user).data
                }
            }

            return make_response(jsonify(response_object)), 201
`

It's pretty verbose so it doesn't always get enforced. But yep, need to create them urself, which isn't too bad because it means u can create some pretty specific behaviour. 

As @RubenGeo pointed out, we should probably be doing consistent validation on our API, which this library also supports (https://marshmallow.readthedocs.io/en/stable/)